src/Adapter/Product/Combination/QueryHandler/GetEditableCombinationsListHandler.php

```diff
@@ -108,8 +109,11 @@ public function __construct(
      */
     public function handle(GetEditableCombinationsList $query): CombinationListForEditing
     {
+        $shopId = $query->getShopId();
```

Comment:
Can you please change the command to handle a ShopConstraint instead of a single shop ID. You don't even need to handle the allShops case you can throw an exception with a `Not handled yet` but at least the command constructor is already ok and we won't need to change it and have a breaking change later.